Jetbrains.Annotations/Remotion.ThirdParty.Dependency.Jetbrains.Annotations.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
should we add the source file in this package since we need to touch things anyway when we build?

Reply:
I'm not sure what you mean with "thing we touch when we build", but I'm guessing you mean when we update the jetbrains annotations code in re-motion?

Reply:
Argh, this should have been "things we touch when we create a new version for this package"

when we update the source files in re-motion, we also need to update the annotations package with a new version number. which means we can also paste / update the source file in the repository at the same time (5 seconds more work).